frontend/public/locales/en/translation.json

```diff
@@ -1399,6 +1402,7 @@
   "Flux": "Flux",
   "Follow documentation for more information.": "Follow documentation for more information.",
   "For more details and logs, click on the nodes to open the properties view.": "For more details and logs, click on the nodes to open the properties view.",
+  "For Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper constraints, response action refers to the enforcementAction.\": \"For Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper constraints, response action refers to the enforcementAction.": "For Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper constraints, response action refers to the enforcementAction.\": \"For Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper constraints, response action refers to the enforcementAction.",
```

Comment:
It looks like the translation still needs to be fixed so that the sentence is duplicated.

Reply:
I think i18n is making this issue. I should change something else